.projenrc.ts

```diff
@@ -107,6 +107,9 @@ new TextFile(project, 'sonar-project.properties', {
     'sonar.sources=./src',
     'sonar.tests=./test',
     'sonar.test.inclusions=**/*.test.*',
+    'sonar.issue.ignore.multicriteria=e1',
```

Comment:
question: Do the Sonar settings have to be duplicated in `.projenrc.ts` and `sonar-project.properties`?